lineage/figures/figureCommon.py

```diff
@@ -151,6 +154,15 @@ def figureMaker(ax, x, paramEst, accuracies, tr, pii, paramTrues, xlabel="Number
     of various types and sizes.
     """
     i = 0
+    ax[i].axis('off')
+
+    i+=1
+    ax[i].axis('off')
+
+    i+=1
+    ax[i].axis('off')
+    
+    i+=1
```

Comment:
In case you were wondering, this is how you set the axis off @jclagarde

Reply:
It has to be done where the axis is being changed in some way (or you have to do it inside the function where the axes are being manipulated).